Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.6-alpine
+
+COPY requirements.txt /app/
+
+RUN true \
+	&& pip install gunicorn \
+	&& pip install -r /app/requirements.txt \
+	&& true
+
```

Comment:
there is no reason to not have the app in the container. the mount has priority over it.